Lib/inspect.py

```diff
@@ -2457,7 +2457,7 @@ class Parameter:
         The annotation for the parameter if specified.  If the
         parameter has no annotation, this attribute is set to
         `Parameter.empty`.
-    * kind : str
+    * kind : int
```

Comment:
It's an `Enum` to be little more precise and also got `description` attribute in 3.8 to give a short description. Relevant typeshed definition : https://github.com/python/typeshed/blob/8e7c32846fc2b07eff10296f815f053a5f6daaa5/stdlib/3/inspect.pyi#L147 . I am not sure about the best way to change this to give a precise type annotation. Any suggestions : @1st1 @pablogsal